src/pydvl/valuation/samplers/powerset.py

```diff
@@ -369,6 +368,95 @@ def _generate(self, indices: IndexSetT) -> SampleGenerator:
                 yield Sample(idx, subset)
 
 
+class OwenSampler(StochasticSamplerMixin, PowersetSampler):
+    """A sampler for Owen shapley values.
+
```

Comment:
Do you think it makes sense to add a small mathematical expression about how the sampling works here?